oaknational/Oak-Web-Application
# feat: use next future image

## Description

- updates to `next/future/image`
- very cool as the old one was rubbish
- also wraps it in `styled` with `BoxProps`, so has the convenience style props which are responsive hurray

## Issue(s)

Fixes #723 
Fixes #722

## How to test

1. Go to {cloud link}
2. Site should be the same as it was
3. however you should see the blog images don't have blue lines under them



## Checklist
